faq/serverless-containers.mdx

```diff
@@ -136,72 +139,80 @@ Ensure that your code avoids heavy computations or long-running initialization a
+### Can I whitelist the IPs of my containers?
+
+Serverless Containers does not yet support Private Networks. However, you can use the Scaleway IP ranges defined at [https://www.scaleway.com/en/peering/](https://www.scaleway.com/en/peering/) on Managed Databases and other products that allow IP filtering.
 
-## Which protocols are supported by Serverless Containers?
+### Which protocols are supported by Serverless Containers?
 
 Serverless Containers use the **http1** protocol by default, but some services (e.g., gRPC) only support http2.
 
 Protocol switching is available in the Console under the `Advanced options` section in the `Deployment` tab.
 
-## Why does my gRPC container not respond?
+### Why does my gRPC container not respond?
 
 Containers use http1 by default, yet the gRPC protocol requires http2. You can upgrade the protocol to http2 (`h2c`).
 
-## How does Serverless Container healthcheck work ?
-
-A Serverless Container is set to `ready` once the specified port is correctly bound to the container, and will start receiving traffic. If your application needs to perform some tasks before receiving traffic (e.g. connect to a database), it's important to run them before binding to the port (starting the webserver).
-For now, the `HEALTHCHECK` Docker directive has no impact on container readiness. In the future, the healthcheck will be customizable for your applications.
-
-## How can I configure access to a Private Network?
+### How can I configure access to a Private Network?
 
 Scaleway Serverless Containers does not currently support Scaleway VPC or Private Networks, though this feature is under development.
 
 To add network restrictions on your resource, consult the [list of prefixes used at Scaleway](https://www.scaleway.com/en/peering/). Serverless resources do not have dedicated or predictable IP addresses.
 
-## How can I attach Block Storage to a Serverless Container?
+### How can I attach Block Storage to a Serverless Container?
 
 Scaleway Serverless Containers do not currently support attaching Block Storage. These containers are designed to be
 stateless, meaning they do not retain data between invocations. For persistent storage, we recommend using external
 solutions like Scaleway Object Storage.
 
-## Why does my container have an instance running after deployment, even with min-scale 0?
-
-Currently, a new container instance will always start after each deployment, even if there is no traffic and the minimum
-scale is set to 0. This behavior is not configurable at this time.
-
-## How can I store data in my Serverless resource?
+### How can I store data in my Serverless resource?
 
 Serverless resources are by default [stateless](/serverless/containers/concepts/#stateless), local storage is ephemeral.
 
 For some use cases, such as saving analysis results, exporting data etc., it can be important to save data. Serverless resources can be connected to other resources from the Scaleway ecosystem for this purpose:
```

Comment:
```suggestion
For certain use cases, such as saving analysis results or exporting data, it can be important to have permanent storage to save data. Serverless resources can be connected to other resources from the Scaleway ecosystem for this purpose:
```